backend/src/main/java/com/ody/mate/service/MateService.java

```diff
@@ -19,15 +16,17 @@ public class MateService {
     private final MateRepository mateRepository;
 
     @Transactional
-    public Mate save(MateSaveRequest mateSaveRequest, Meeting meeting, Member member) {
-        if (mateRepository.existsByMeetingIdAndNicknameNickname(meeting.getId(), mateSaveRequest.nickname())) {
+    public Mate save(Mate mate) {
+        if (mateRepository.existsByMeetingIdAndNicknameNickname(
```

Comment:
[제안]
NicknameNickname으로 메서드 네이밍을 한 이유가 궁금합니다! 확실하진 않지만 Embeddable이면 Nickname만으로 조회가 가능하지 않을까 싶어서 궁금해졌어요🤔 

```suggestion
        if (mateRepository.existsByMeetingIdAndNickname(
```

Reply:
디미터 법칙을 지키기 위해 getNickname()를 따로 정의했기 때문인 것 같아요.

Reply:
`existsByMeetingIdAndNickname` 네이밍 쿼리를 사용했을 때
임베디드 클래스인 Nickname에 접근하는데 String으로 넘긴 nickname과 Nickname 객체와 타입이 일치하지 않아서 
`InvalidDataAccessApiUsageException` 예외가 발생해요
그래서 String으로 넘긴 nickname과 타입이 일치하는 Nickname객체의 내부 필드 값을 네이밍 쿼리로 작성해야 하더라구요 👍🏻